test/uk/gov/hmrc/gform/auth/AuthServiceSpec.scala

```diff
@@ -128,9 +131,12 @@ class AuthServiceSpec extends ExampleData with SpecWithFakeApp {
   val authConfigEnrolment = HmrcAgentWithEnrolmentModule(RequireMTDAgentEnrolment, enrolmentAuthCheck)
   val formTemplateEnrolment = formTemplate.copy(authConfig = authConfigEnrolment)
 
-  val authEeitt = EeittModule(RegimeId("TT"))
+  val authEeitt: AuthConfig = EeittModule(RegimeId("TT"))
   val formTemplateEeitt = formTemplate.copy(authConfig = authEeitt)
 
+  val authAWSALB: AuthConfig = AWSALBAuth
+  val formTemplateAWSALBMissingHTTPHeader = formTemplate.copy(authConfig = authAWSALB)
```

Comment:
This `MissingHTTPHeader` part of the name seems unnecessary and misleading

Reply:
Yes that's incorrectly named